devDocs/userGuideStandards.md

```diff
@@ -0,0 +1,26 @@
+# User Guide Standards
+This document aims to create a standard style guide for writing documentation in the User Guide.
+
+The principles outlined in ["The Documentation System" guide for reference materials](https://documentation.divio.com/reference/) are encouraged when working on the User Guide and this document.
+
+## General standards
+- Key commands should be lowerCamelCase and encapsulated in monospace code-block formatting.
+  - e.g. ` ``NVDA+control+upArrow`` `

```

Comment:
But in this example, `NVDA` is uppercase, not lowerCamelCase. Should it be `nvda` (lower case) or should `NVDA` be an exception (that should be documented)?

Reply:
The casing of initialisms/acronyms is not well defined for camel casing.
I hope that the example given is sufficient documentation of the casing of NVDA.

Reply:
You could add an explicit point that the `NVDA` key is always uppercase.